ts/lib/auth/computeclient.ts

```diff
@@ -102,7 +116,7 @@ export default class Compute extends Auth2Client {
           err.message = helpfulMessage;
         } else {
           err = new Error(helpfulMessage);
```

Comment:
Is it possible to directly create a `RequestError` here to avoid the cast later?